frontend/apps/web/app/(mgmt)/[account]/transformers/EditTransformerOptions.tsx

```diff
@@ -216,3 +232,11 @@ function NoAdditionalTransformerConfigurations(): ReactElement {
     </Alert>
   );
 }
+
+export function constructDocsLink(source: number): string {
+  const name: string = TransformerSource[source]
+    .toLowerCase()
+    .replaceAll('_', '-');
```

Comment:
we do have this function in the utils `getTransformerSourceString` that effectively does this (minute the replace all). Not sure if that is useful here.

Reply:
yeah we need the replace since the link sin the docs has dashes while the transformer source has underscores